File: pset1/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>
int main(void)
{
    float change;
    do{
        change = get_float("Para üstünü giriniz.\n");
    }while(change <= 0);
    int i = 0;
    int change1 = round(change*100);
    while(change1 > 0)
    {
      
        if(change1 >= 25)
        {change1 = change1 - 25;
        i++;}
        
        if(change1 >= 10 && change1 < 25){
        change1 = change1 - 10;
        i++;}
        
        if(change1 >= 5 && change1 < 10)
        {change1 = change1 - 5;
        i++;}
       
        if(change1 >= 1 && change1 < 5)
        {change1 = change1 - 1;
        i++;}
        
        
    }
   printf("%d\n", i);
    return 0;
}

File: pset6/cash/cash.py
from cs50 import get_float

while True:
    
    money = get_float("Para Üstü: ")
    
    if money > 0:
        break
money = round(money * 100)
count = 0
    
while money > 0:
    
    if money >= 25:
        
        money -= 25
        count += 1
        
    elif money >= 10 and money < 25:
        
        money -= 10
        count += 1
        
    elif money >= 5 and money < 10:
        
        money -= 5
        count += 1
        
    else:
        
        money -= 1
        count += 1
        
print(count)